packages/derive/src/lib.rs

```diff
@@ -86,3 +88,333 @@ pub fn entry_point(_attr: TokenStream, mut item: TokenStream) -> TokenStream {
     item.extend(entry);
     item
 }
+
```

Comment:
Please note to it when reviewing.
- proc_macro must be defined in lib.rs (crate root).
- proc_macro cannot support rust test.
  - Instead, dynamic-callee-contract and caller-contract's integration-test cover them.